WordPress/src/main/java/org/wordpress/android/ui/accounts/login/Login2FaFragment.java

```diff
@@ -116,6 +117,7 @@ protected void onHelp() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        ((WordPress) getActivity().getApplication()).component().inject(this);
```

Comment:
If we are going to inject for every child of `LoginBaseFormFragment`, I think it'd be better if we just did the inject in that base class. This way we can remove all these injects and slim down the `AppComponent` class a little bit.